ansible/lemmy.yml

```diff
@@ -24,10 +24,11 @@
       creates: '/etc/letsencrypt/live/{{domain}}/privkey.pem'
 
   - name: create lemmy folder
-    file: path={{item.path}} state=directory
+    file: path={{item.path}} {{item.owner}} state=directory
```

Comment:
Just noticed, this one needs to be `owner={{item.owner}}`.

Reply:
Fixed.